### PR TITLE
CSS layout cookbook: contribute page

### LINKED ISSUE #184: Update CSS Cookbook
### Problem statement

The Cookbook section is outdated, with indicators that well supported features are not fully supported yet.

### Proposed solutions

Update ALL. THE. THINGS.

### Task list

Upated the following pages and removed browser compact (except baseline, if present)

- [X] [Media objects](/en-US/docs/Web/CSS/Layout_cookbook/Media_objects)   https://github.com/mdn/content/pull/30142
- [x] [Columns](/en-US/docs/Web/CSS/Layout_cookbook/Column_layouts) https://github.com/mdn/content/pull/29961
- [x] [Center an element](/en-US/docs/Web/CSS/Layout_cookbook/Center_an_element) (https://github.com/mdn/content/pull/29609)
- [X] [Sticky footers](/en-US/docs/Web/CSS/Layout_cookbook/Sticky_footers)  https://github.com/mdn/content/pull/30146
- [X]  [Split navigation](/en-US/docs/Web/CSS/Layout_cookbook/Split_Navigation)   https://github.com/mdn/content/pull/30145
- [x]  [Breadcrumb navigation](/en-US/docs/Web/CSS/Layout_cookbook/Breadcrumb_Navigation)  (https://github.com/mdn/content/pull/29921)
- [x]  [List group with badges](/en-US/docs/Web/CSS/Layout_cookbook/List_group_with_badges) 
- [X] [Pagination](/en-US/docs/Web/CSS/Layout_cookbook/Pagination)  https://github.com/mdn/content/pull/30143/
- [X] [Card](/en-US/docs/Web/CSS/Layout_cookbook/Card) https://github.com/mdn/content/pull/29972
- [x] [Grid wrapper](/en-US/docs/Web/CSS/Layout_cookbook/Grid_wrapper)  https://github.com/mdn/content/pull/29998  

Also, 
- [x] [Contribute a recipe](/en-US/docs/Web/CSS/Layout_cookbook/Contribute_a_recipe)   https://github.com/mdn/content/pull/29993

### PR DESCRIPTION
**Pull request by estelle**
Updated the how to contribute to the MDN cookbook. 
Fixes openwebdocs/project#184.

### CODE REVIEW

**Comment by dipikabh** [comment]
Apologies, @estelle. I haven't been able to review this yet. Since I'll be out of office next week, I don't want to hold you up with a review in between if I start now. It might be best to have someone else take a look.

File: files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md [hunk 1]
```diff
@@ -12,69 +12,82 @@ If you would like to contribute an example for the Layout Cookbook, then this pa
...
 
 ### 5. Create your page
 
-Once your example is merged you can [open a pull request](/en-US/docs/MDN/Community/Pull_requests#open_a_pull_request) to create a new page for it in the [Layout Cookbook directory](https://github.com/mdn/content/tree/main/files/en-us/web/css/layout_cookbook).
+Once your example is merged, you can [open a pull request](/en-US/docs/MDN/Community/Pull_requests#open_a_pull_request) to create a new page in the [Layout Cookbook directory](https://github.com/mdn/content/tree/main/files/en-us/web/css/layout_cookbook).
 There is [a template for Cookbook pages](https://github.com/mdn/content/blob/main/files/en-us/web/css/layout_cookbook/contribute_a_recipe/cookbook_template/index.md?plain=1) that you should use.
 The template explains what should be part of each section, and you can refer to other cookbook examples for more help.
 
-If you need to ask any questions, or want someone to take a look at your page, get in touch with us on any of our [communication channels](/en-US/docs/MDN/Community/Communication_channels).
+Also, add a link to the new recipe in the top-level [Layout Cookbook page](/en-us/web/css/layout_cookbook/).
```
Comment (mathewhodson) [inline files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md:88]: The bot is reporting a link flaw because of the trailing slash.

Comment (estelle) [inline files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md:88]:
```suggestion
Also, add a link to the new recipe in the top-level [Layout Cookbook page](/en-us/web/css/layout_cookbook).
```

**Comment by dipikabh** [comment]
Am back and looking at this now 🙂

File: files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md [hunk 2]
```diff
@@ -12,69 +12,82 @@ If you would like to contribute an example for the Layout Cookbook, then this pa
...
+
+It can be handy to include the body CSS rules that are in our included stylesheet. Take a look at [`cookbook-template--download.html`](https://github.com/mdn/css-examples/blob/main/css-cookbook/cookbook-template--download.html); these are included as a starting point.
 
 ### 4. Make a Pull Request with your example
 
-Make a PR on the [CSS Examples repo](https://github.com/mdn/css-examples). This way we can help you with any changes to the example that may be needed before you create your page, plus the example will need to be live to be included some it makes sense to create it first. Include in your PR an explanation of the pattern and what you are demonstrating.
+Make a [PR](/en-US/docs/MDN/Community/Pull_requests#open_a_pull_request) on the [CSS Examples repo](https://github.com/mdn/css-examples). This way, we can help you with any changes to the example that may be needed before you create your page. Plus, the example will need to be live to be included. This is why it makes sense to create the code examples first, before the explanatory guide. In your PR, explain the pattern and what you are demonstrating.
```
Comment (dipikabh) [inline files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md:80]: Shouldn't the PR link here point to the PRs in that repo? If we want, we can add a separate sentence about general guidelines for opening pull requests
```suggestion
Open a [pull request (PR)](https://github.com/mdn/css-examples/pulls) on the [CSS Examples repo](https://github.com/mdn/css-examples). This allows us to help you with any changes to the example that may be needed before you create your page. Plus, the example will need to be live to be included. This is why it makes sense to first create the code examples, and then the explanatory guide. In your PR, explain the pattern and what you are demonstrating.
```

File: files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md [hunk 3]
```diff
@@ -12,69 +12,82 @@ If you would like to contribute an example for the Layout Cookbook, then this pa
...
 
 ### 5. Create your page
 
-Once your example is merged you can [open a pull request](/en-US/docs/MDN/Community/Pull_requests#open_a_pull_request) to create a new page for it in the [Layout Cookbook directory](https://github.com/mdn/content/tree/main/files/en-us/web/css/layout_cookbook).
+Once your example is merged, you can [open a pull request](/en-US/docs/MDN/Community/Pull_requests#open_a_pull_request) to create a new page in the [Layout Cookbook directory](https://github.com/mdn/content/tree/main/files/en-us/web/css/layout_cookbook).
```
Comment (dipikabh) [inline files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md:84]:
```suggestion
After your example PR is merged, open a pull request to create a new page in the [Layout Cookbook directory](https://github.com/mdn/content/tree/main/files/en-us/web/css/layout_cookbook). See our [guidelines for opening a pull request](/en-US/docs/MDN/Community/Pull_requests#open_a_pull_request).
```

File: files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md [hunk 4]
```diff
@@ -12,69 +12,82 @@ If you would like to contribute an example for the Layout Cookbook, then this pa
...
 There is [a template for Cookbook pages](https://github.com/mdn/content/blob/main/files/en-us/web/css/layout_cookbook/contribute_a_recipe/cookbook_template/index.md?plain=1) that you should use.
 The template explains what should be part of each section, and you can refer to other cookbook examples for more help.
 
-If you need to ask any questions, or want someone to take a look at your page, get in touch with us on any of our [communication channels](/en-US/docs/MDN/Community/Communication_channels).
+Also, add a link to the new recipe in the top-level [Layout Cookbook page](/en-us/web/css/layout_cookbook).
+
+If you have any questions or want someone to take a look at your page, get in touch via any [communication channel](/en-US/docs/MDN/Community/Communication_channels).
```
Comment (dipikabh) [inline files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md:90]:
```suggestion
If you have any questions or want someone to take a look at your page, get in touch via any of our [communication channels](/en-US/docs/MDN/Community/Communication_channels).
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for updating this page, @estelle. I've included some edit suggestions.

**Comment by estelle** [comment]
Welcome back @dipikabh and thank you for the review.
I made all the suggested changes.

Thanks!

File: files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md [hunk 5]
```diff
@@ -12,69 +12,82 @@ If you would like to contribute an example for the Layout Cookbook, then this pa
...
-   3. The choices made
-   4. Any useful fallbacks or alternative methods
-   5. Any accessibility concerns
-   6. Browser compatibility
-   7. Additional resources
+1. [Build a pattern](#1-build-a-pattern)
+2. [Create a live example](#2-create-a-live-example)
+3. [Create a downloadable version](#3-create-a-downloadable-version)
+4. [Open a pull request](#4-open-a-pull-request-with-your-example)
+5. [Add your content to MDN](#5-create-your-page)
```
Comment (dipikabh) [inline files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md:38]: Checked these links in the local preview. The hyphens need to be underscores. Plus there is a period after the numbers in the headings

```suggestion
1. [Build a pattern](#1._build_a_pattern)
2. [Create a live example](#2._create_a_live_example)
3. [Create a downloadable version](#3._create_a_downloadable_version)
4. [Open a pull request](#4._open_a_pull_request_with_your_example)
5. [Add your content to MDN](#5._create_your_page)
```

File: files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md [hunk 6]
```diff
@@ -12,69 +12,82 @@ If you would like to contribute an example for the Layout Cookbook, then this pa
...
+It can be handy to include the body CSS rules from our included stylesheet. Take a look at [`cookbook-template--download.html`](https://github.com/mdn/css-examples/blob/main/css-cookbook/cookbook-template--download.html) as a guide; these rules are provided as a starting point.
 
-Make a PR on the [CSS Examples repo](https://github.com/mdn/css-examples). This way we can help you with any changes to the example that may be needed before you create your page, plus the example will need to be live to be included some it makes sense to create it first. Include in your PR an explanation of the pattern and what you are demonstrating.
+### 4. Open a pull request with your example
+
+Open a [pull request (PR)](/en-US/docs/MDN/Community/Pull_requests#open_a_pull_request) on the [CSS Examples repo](https://github.com/mdn/css-examples/pulls). This allows us to help you with any changes to the example that may be needed before you create your page. Plus, the example will need to be live to be included. This is why it makes sense first to create the code examples, and then the explanatory guide. In your PR, explain the pattern and what you are demonstrating.
```
Comment (dipikabh) [inline files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md:80]: - Removed extra line
- Updated this sentence "This is why it makes sense to first create the code examples and then the explanatory guide."
- Moving the PR link to the end
```suggestion
Open a pull request (PR) on the [CSS Examples repo](https://github.com/mdn/css-examples/pulls). This allows us to help you with any changes to the example that may be needed before you create your page. Plus, the example will need to be live to be included. This is why it makes sense to first create the code examples and then the explanatory guide. In your PR, explain the pattern and what you are demonstrating. See our [guidelines for opening a pull request](/en-US/docs/MDN/Community/Pull_requests#open_a_pull_request).
```

File: files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md [hunk 7]
```diff
@@ -12,69 +12,82 @@ If you would like to contribute an example for the Layout Cookbook, then this pa
...
 
 ### 5. Create your page
 
-Once your example is merged you can [open a pull request](/en-US/docs/MDN/Community/Pull_requests#open_a_pull_request) to create a new page for it in the [Layout Cookbook directory](https://github.com/mdn/content/tree/main/files/en-us/web/css/layout_cookbook).
-There is [a template for Cookbook pages](https://github.com/mdn/content/blob/main/files/en-us/web/css/layout_cookbook/contribute_a_recipe/cookbook_template/index.md?plain=1) that you should use.
-The template explains what should be part of each section, and you can refer to other cookbook examples for more help.
+After your example PR is merged, open a pull request to create a new page in the [Layout Cookbook directory](https://github.com/mdn/content/tree/main/files/en-us/web/css/layout_cookbook). See our [guidelines for opening a pull request](/en-US/docs/MDN/Community/Pull_requests#open_a_pull_request).
```
Comment (dipikabh) [inline files/en-us/web/css/layout_cookbook/contribute_a_recipe/index.md:84]: Moving PR guideline link to the previous step
```suggestion
After your example PR is merged, open a pull request to create a new page in the [Layout Cookbook directory](https://github.com/mdn/content/tree/main/files/en-us/web/css/layout_cookbook).
```

**Review by dipikabh — APPROVED** [review]
Thanks! A few more changes are needed. But its looking in good shape, so leaving my approval. Please feel free to merge after the updates.

**Comment by dipikabh** [comment]
Argh, my bad for removing the required blank line! Thanks for fixing @teoli2003.

**Pull request merged**